e2e-tests/adapters/cypress/e2e/basics.cy.ts

```diff
@@ -1,44 +1,49 @@
 import { title } from "../../constants"
 
-describe('Basics', () => {
+describe("Basics", () => {
   beforeEach(() => {
     cy.intercept("/gatsby-icon.png").as("static-folder-image")
-    cy.intercept("/static/astro-**.png").as("img-import")
+    cy.intercept("/static/astro-**.png", req => {
+      req.on("before:response", res => {
+        // this generally should be permamently cached, but that cause problems with intercepting
+        // see https://docs.cypress.io/api/commands/intercept#cyintercept-and-request-caching
+        // so we disable caching for this response
+        // tests for cache-control headers should be done elsewhere
 
-    cy.visit('/').waitForRouteChange()
+        res.headers["cache-control"] = "no-store"
+      })
+    }).as("img-import")
+
+    cy.visit("/").waitForRouteChange()
```

Comment:
This is only real change in test - rest of it is just autoformatting (yikes). Apperently with `ntl serve` permanent caching was really happing and test setup is kind of meh when that happens ( https://docs.cypress.io/api/commands/intercept#cyintercept-and-request-caching ), so this is just workaround to get test working (and probably note to actually add caching behavior tests in the future PR)